tests/xharness/ProjectFileExtensions.cs

```diff
@@ -598,6 +598,17 @@ public static IEnumerable<string> GetExtensionProjectReferences (this XmlDocumen
 					yield return node.Attributes ["Include"].Value;
 			}
 		}
+
+		public static IEnumerable<string> GetNunitAndXunitTestReferences (this XmlDocument csproj)
+		{
+			var nodes = csproj.SelectNodes ("//*[local-name() = 'Reference']");
+			foreach (XmlNode node in nodes) {
+				var includeValue = node.Attributes ["Include"].Value;
+				if (includeValue.EndsWith ("_test.dll") || includeValue.EndsWith ("_xunit-test.dll"))
```

Comment:
`.EndsWith (..., StringComparison.Ordinal)` is faster (or `.OrdinalIgnoreCase` if that's desired)